### PR TITLE
init branch

### PR DESCRIPTION
**Pull request by smoreg**
First backbone of project

### CODE REVIEW

File: internal/app/handler/base.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package handler
+
+import (
+	"backend/internal/repo"
```
Comment (Ketarica) [inline internal/app/handler/base.go:4]: lox perepeshi

File: internal/app/handler/base.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	shortener "backend/internal/service"
+)
+
+type Handler struct {
+	Service shortener.Service
```
Comment (Ketarica) [inline internal/app/handler/base.go:9]: i tyt toje

**Review by Ketarica — APPROVED** [review]
ok

File: internal/app/handler/base.go (hunk 1 above)
Comment (smoreg) [reply internal/app/handler/base.go:4]: no

**Pull request merged**